### PR TITLE
[WIP] - Beyond Living Styleguides

### PR DESCRIPTION
**Pull request by eschaefer**
- [ ] Add illustration from Paul Woods

### CODE REVIEW

**Comment by eschaefer** [comment]
ping @edenspiekermann/js for some extra eyeballs! Let me know if anything is missing, wrong, or could be added.

**Comment by eschaefer** [comment]
ping @edenspiekermann/devs for some extra eyeballs! Let me know if anything is missing, wrong, or could be added.

File: _posts/2016-08-18-beyond-living-styleguides.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+
+## Component libraries
+
+Many of our projects are already built with a component-based approach. This is a wonderful side-effect of using [React](https://facebook.github.io/react/). All UI elements in a well-built React application end up being self-encapsulated (markup and styles bundled together) and composable. This simple fact has far-reaching implications:
+
+1. We can import all our components and render each one into a library.
+2. We can minimize the chances of dead code and designs.
+3. We can eliminate duplicated effort to keep production elements in sync with the styleguide.
+4. We can auto-document how often and where a component is used.
+
```
Comment (nburkley) [inline _posts/2016-08-18-beyond-living-styleguides.md]: maybe these should be bullet point - there's no order here?

File: _posts/2016-08-18-beyond-living-styleguides.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+> BlueKit automatically generates a library from your React components with editable props and live preview.
+
+This one is seems like an underdog in terms of Github stars, but looks really polished.
+
+![BlueKit](https://cloud.githubusercontent.com/assets/547148/17784131/2c448104-6530-11e6-8efb-a01c2071cef7.png)
+
+
+### [Atellier](http://scup.github.io/atellier)
+
```
Comment (nburkley) [inline _posts/2016-08-18-beyond-living-styleguides.md]: Worth adding a line on this one as well?

**Comment by nburkley** [comment]
looks great @eschaefer

File: _posts/2016-08-18-beyond-living-styleguides.md (hunk 1 above)
Comment (eschaefer) [inline _posts/2016-08-18-beyond-living-styleguides.md]: yep, good point

File: _posts/2016-08-18-beyond-living-styleguides.md (hunk 2 above)
Comment (eschaefer) [inline _posts/2016-08-18-beyond-living-styleguides.md]: done!

**Comment by eschaefer** [comment]
Keeping this on hold, since it will likely be going into Smashing Mag instead.

**Pull request closed without merging**